src/visualization/interpolate.jl

```diff
@@ -250,6 +250,100 @@ function unstructured_2d_to_1d(original_nodes, unstructured_data, nvisnodes, sli
   return get_data_1d(reshape(new_nodes[:, 1:new_id], 1, n_nodes_in, new_id), new_unstructured_data[:, 1:new_id, :], nvisnodes)
 end
 
+# Convert 2d unstructured data to 1d data at given coordinates.
+function unstructured_2d_to_1d_along_curve(original_nodes, unstructured_data, nvisnodes, coordinates, mesh, solver, cache)
+
+  @assert size(coordinates) == (2, size(coordinates)[2]) "Coordinates along curve must be 2xn dimensional."
+
+  n_elements_curve = size(coordinates)[2]
```

Comment:
I wouldn't call this `elements`, since this name is tightly coupled to DG elements in Trixi. Maybe just `points`?

Reply:
Done